airbyte-db/lib/src/main/resources/configs_database/schema2.sql

```diff
@@ -0,0 +1,218 @@
+CREATE
+    TYPE "source_type" AS enum(
+        'api',
+        'file',
+        'database',
+        'custom'
+    );
+
+CREATE
+    TYPE "actor_type" AS enum(
+        'source',
+        'destination'
+    );
+
+CREATE
+    TYPE "operator_type" AS enum(
+        'normalization',
+        'dbt'
+    );
+
+CREATE
+    TYPE "namespace_definition_type" AS enum(
+        'source',
+        'destination',
+        'customformat'
+    );
+
+CREATE
+    TYPE "status_type" AS enum(
+        'active',
+        'inactive',
+        'deprecated'
+    );
+
+CREATE
+    TABLE
+        IF NOT EXISTS "workspace"(
+            id uuid NOT NULL,
+            customer_id uuid NULL,
+            name VARCHAR(256) NOT NULL,
+            slug VARCHAR(256) NOT NULL,
+            email VARCHAR(256) NULL,
+            initial_setup_complete BOOLEAN NOT NULL,
+            anonymous_data_collection BOOLEAN NULL,
+            news BOOLEAN NULL,
+            security_updates BOOLEAN NULL,
+            display_setup_wizard BOOLEAN NULL,
+            tombstone BOOLEAN NOT NULL DEFAULT FALSE,
+            notifications jsonb NULL,
+            first_completed_sync BOOLEAN NULL,
+            feedback_done BOOLEAN NULL,
+            created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CAST(
+                CURRENT_TIMESTAMP AS TIMESTAMP WITH TIME ZONE
+            ),
+            updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CAST(
+                CURRENT_TIMESTAMP AS TIMESTAMP WITH TIME ZONE
+            ),
+            PRIMARY KEY(id)
+        );
+
+CREATE
+    TABLE
+        IF NOT EXISTS "actor_definition"(
+            id uuid NOT NULL,
+            name VARCHAR(256) NOT NULL,
+            docker_repository VARCHAR(256) NOT NULL,
+            docker_image_tag VARCHAR(256) NOT NULL,
+            documentation_url VARCHAR(256) NULL,
+            icon VARCHAR(256) NULL,
+            actor_type actor_type NOT NULL,
+            source_type source_type NULL,
```

Comment:
this field shouldn't belong here since it is only for sources

Reply:
agreed. this is a conscious choice that @subodh1810 and I made. This is the only difference between Source and Destination Definition right now. As they drift further apart we will do a proper polymorphic association. In the meanwhile adding that complexity for this field felt like the wrong tradeoff. 

@subodh1810 we should add comment (wherever possible, i know it's probably not possible in this dump file) explaining this tradoeff.